### ISSUE TITLE
Group Blocks Inside Reusable Blocks: Top-Level Cannot Be Edited, Sub-Level Removed from Tab Order

### ISSUE THREAD

**Issue by 0aveRyan** [WordPress/gutenberg#19373]
**Describe the bug**
Inside of Reusable Blocks, Group Blocks cannot be accessed by keyboard navigation for top-level or nested Group Blocks. While nested Group Blocks can be clicked into, top-level Group Blocks inside a Reusable Block cannot be clicked or accessed by keyboard. 

As noted in #19372, the Block Navigation doesn't reveal the contents of Reusable Blocks, so it's impossible to edit Group Block settings at the top-level and quite challenging to edit nested Group Blocks.

**To reproduce**
1. Insert a Group Block (recommend giving a Background Color for visibility)
2. Insert a Paragraph inside the Group Block.
3. Insert a nested Group Block inside the Group Block (recommend giving a different Background Color)
4. Insert a Paragraph inside the nested Group Block.
5. Attempt to access Group Block settings via TAB/Shift+TAB keyboard navigation.
6. Attempt to access Group Block settings via mouse clicks (only a mouse click in the upper right corner of the nested Group Block, outside the Paragraph block boundary will trigger the toolbar and Block selection. No way to click or TAB into the top-level Group Block)

_Worth noting, this behavior doesn't occur inside Reusable Blocks with Columns or other blocks containing child blocks. This behavior only happens inside of Reusable Blocks, not when Group blocks are inserted into normal Post Content._

**Expected behavior**
Group Blocks need to be able to be tabbed into for a11y and general usability reasons. Group Blocks should also be selectable via mouse click, as this is likely to be the more common use case. Ideally, Reusable Blocks' contents should be shown in the Block Navigation (#19372) as a reliable way to select Groups and trigger toolbar/sidebar to initialize.

**Screenshots**
![Screen Shot 2019-12-31 at 8 55 37 AM](https://user-images.githubusercontent.com/9565066/71626623-7032c980-2bab-11ea-9ab7-85630f8f2e0e.png)

WordPress: Version 5.3.3-alpha-46995

**Comment by paaljoachim** [comment]
Hey @0aveRyan David

Thanks for creating the issue!
Is this issue still valid?

There is a general cleanup of Reusable blocks happening lately. Here is the general issue.
: https://github.com/WordPress/gutenberg/issues/27890

**Comment by paaljoachim** [comment]
Testing using WordPress 5.7.1. Twenty Twenty One. Gutenberg plugin 10.5.

Focusing on mouse clicks:

https://user-images.githubusercontent.com/5323259/116604734-cdb96080-a92e-11eb-9edc-245b89ad7ffb.mp4

I am able to select both Group block settings.

**Comment by paaljoachim** [comment]
Could you Joe @joedolson or Stefano @Ryokuhi check to see if this issue is still valid?
Thank you!

**Comment by shikkaba** [comment]
I wonder if this is related to the glitch I'm having where you can't see content inside a nested group block unless you switch preview from desktop.

Not reporting it here, it just seems to be the same block stack that is causing problems.

**Comment by jordesign** [comment]
Seeing no confirmation this is still an issue - I'm going to close it for now. It if _does_ prove to be a problem still we can re-open it.
